atlasdb-perf/src/main/java/com/palantir/atlasdb/performance/benchmarks/table/WideRowTable.java

```diff
@@ -40,9 +40,7 @@
  * State class for creating a single Atlas table with one wide row.
  */
 @State(Scope.Benchmark)
```

Comment:
Yeah I was wondering that too, but decided to be consistent with the existing code (for example, see RegeneratingTable - it is an abstract base class but is marked with the annotation nonetheless)